apps/dashr-clustergram/app.R

```diff
@@ -41,13 +41,12 @@ header <- htmlDiv(
     htmlA(
       id = "dashbio-logo",
       children = list(
-        htmlImg(
-          src = "assets/dashbio_logo_transparent.png" 
-        )
+        htmlImg(src='assets/plotly-dash-bio-logo.png', height = '36', width = '180',
+                style = list('top' = '10', 'margin-left' = '10px'))
       ),
       href = "/Portal"
     ),
-    htmlH2("Dash Clustergram"),
+    htmlH2("    Clustergram"),
```

Comment:
Why is there still whitespace here?

Reply:
I left in the whitespace to act as some spacing from the logo so it's not side-by-side. Another way to achieve the same thing would be to add padding to either container. Would that be a better alternative?

Reply:
Ah I see! Yes, in this case we should be using CSS. Otherwise, it isn't super clear what you're trying to do.

Reply:
Fixed!